example/package.json

```diff
@@ -23,9 +23,9 @@
   },
   "dependencies": {
     "@mia-platform/lc39": "mia-platform/lc39",
-    "fastify-env": "^0.6.1"
+    "fastify-env": "^1.0.1"
   },
   "engines": {
-    "node": ">=8"
```

Comment:
This is a breaking change. Is there a reason for that change? For example a feature of nodejs that is not present in 8.

Reply:
I honestly mindlessly took the change from previous PR #4 [package.json](https://github.com/mia-platform/lc39/pull/4/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json#diff-012ac553d93d987b8d34ad013052bc96)

I guess we can keep support to node 8 I don't see the problem, maybe we should add CI with the version to ensure we're still supporting it.

Reply:
I've run the test locally with lts/carbon and everything worked fine so i'm reverting this change